protocol/flatbuffers/rpc.fbs

```diff
@@ -151,6 +154,8 @@ table SetWifiRequest {
 table SerialUpdateResponse {
     log: string;
     closed: bool;
+	rts: bool;
```

Comment:
Formatting should use 4 spaces for indentation (for now, until we switch to tabs)